app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>show the posts of a user</h1>
\ No newline at end of file
+<section class="show-user">
+  <h1>Show all Posts of a user</h1>
+
+  <% render 'users/user', user: @user %>
+
+   <article class="user d-flex">
+    <div class="img-container">
+      <img src="<%= @user.photo %>" alt="User_Photo">
+    </div>
+    <div class="details d-flex">
+      <h2 class="name"><%= @user.name %></h2>
+      <span class="sub">Number of posts: <%= @user.postsCounter %></span>
```

Comment:
- [ ] @Christabelle143 great work working on the changes requested, however, I couldn't see the posts counter showing the  number of posts, same case to the comments and likes counters
![image](https://user-images.githubusercontent.com/61408860/206862409-eea4efc4-4182-42db-a85c-ef501d3c0b14.png)

Reply:
- [ ] [optional] I couldn't access your show page even when loading the URL manually got this error below 
![image](https://user-images.githubusercontent.com/61408860/206862521-793210f7-d338-4497-8e9f-da7e64cb6cee.png)

Reply:
Hello @danmainah 
**@Christabelle143 great work working on the changes requested, however, I couldn't see the posts counter showing the number of posts, same case to the comments and likes counters**
What you requested as shown above is not part of the requirement. This is to be implemented on the next next project  that's the part for the postCounter, Comment and likesCounter

Reply:
For the optional suggestion just checked it and i can't seem to get that error whenever I load the page
Here are the different pages loading quite well. And the other comment is to be implemented in the next project
![snip1](https://user-images.githubusercontent.com/98928638/206863480-e40863fa-3dcb-40b7-8de1-298c9873c78a.PNG)
![snip3](https://user-images.githubusercontent.com/98928638/206863494-fb2558c3-ad99-4420-b02a-a941ed7e8321.PNG)
![snip4](https://user-images.githubusercontent.com/98928638/206863508-025d25b1-6a67-463e-b7c6-6b5e6cf30b79.PNG)

Reply:
@Christabelle143 let me recheck it and get back at you.